uportal-war/src/main/java/org/jasig/portal/groups/pags/dao/jpa/PersonAttributesGroupTestDefinitionImpl.java

```diff
@@ -103,11 +108,11 @@ public PersonAttributesGroupTestDefinitionImpl(
 
     @ElementCollection(fetch=FetchType.EAGER)
     @CollectionTable(name=JpaPersonAttributesGroupTestDefinitionDao.TABLENAME_PREFIX + "_INCLUDES")
-    private Set<String> includes = Collections.emptySet();
+    private Set<String> includes = new HashSet<>(0);
```

Comment:
Good call.  This approach to defining `includes` and `excludes` is better than what I had -- works much better with the setters.